developer_guides/linux_driver/architecture/sof_driver_arch.rst

```diff
@@ -105,7 +105,7 @@ SOF on both the host and the DSP serializes the sending of their IPC messages. T
 SPI
 ===
 
-IPC messages have the same structure as in the PCI case, but they are sent and received over an SPI bus. The SPI transfer is always initiated by the SPI primary, which is the host. Therefore, the DSP cannot send asynchronous messages to the host using only the SPI bus. To overcome this limitation, an additional GPIO line is used by the DSP to trigger an interrupt on the host to request it to read out an IPC message. Support for such devices is still experimental in SOF. Details will be added later.
+IPC messages have the same structure as in the PCI case, but they are sent and received over an SPI bus. The SPI transfer is always initiated by the SPI owner, which is the host. Therefore, the DSP cannot send asynchronous messages to the host using only the SPI bus. To overcome this limitation, an additional GPIO line is used by the DSP to trigger an interrupt on the host to request it to read out an IPC message. Support for such devices is still experimental in SOF. Details will be added later.
```

Comment:
It may be simpler to just write

"The SPI transfer is always initiated by the host"

Reply:
I will go with that. Thanks you, @plbossart!